Exiled.API/Features/Server.cs

```diff
@@ -248,7 +248,7 @@ public static bool ShutdownRedirect(ushort redirectPort)
         /// </summary>
         /// <param name="command">The command to be run.</param>
         /// <param name="sender">The <see cref="CommandSender"/> running the command.</param>
-        public static void RunCommand(string command, CommandSender sender = null) => GameCore.Console.singleton.TypeCommand(command, sender);
+        public static string RunCommand(string command, CommandSender sender = null) => GameCore.Console.singleton.TypeCommand(command, sender);
```

Comment:
Breaking change make obsolete the previous one also can we get if the command has successfully be made or not ?

Reply:
> Breaking change make obsolete the previous one also can we get if the command has successfully be made or not ?

Compile Error CS0111 will be thrown if a class contains two member declarations with the same name and parameter types.

Reply:
> > Breaking change make obsolete the previous one also can we get if the command has successfully be made or not ?
> 
> 
> 
> Compile Error CS0111 will be thrown if a class contains two member declarations with the same name and parameter types.

Can be renamed to ExecuteCommand or something else like this. It really will be a breaking change so

Reply:
@Misaka-ZeroTwo @louis1706 check please